frontend/src/Users/Users.tsx

```diff
@@ -1,5 +1,173 @@
+        </Select>
+        <FormGroup>
+        <FormControlLabel
+          control={<Checkbox name="isBlocked" />}
+          label={t('users.dialog.block-access')}
+        />
+        </FormGroup>
+        <TextField
+          autoFocus
+          margin="dense"
+          id="fullName"
+          label={t('users.full-name')}
+          type="text"
+          fullWidth
+          variant="standard"
+          value={name}
+          placeholder={t('users.full-name')!!}
+        />
+        <TextField
+          autoFocus
+          margin="dense"
+          id="password"
+          label={t('users.dialog.new-password')}
+          type="password"
+          fullWidth
+          variant="standard"
+          placeholder={t('users.dialog.new-password')!!}
+        />
+        <TextField
+          autoFocus
+          margin="dense"
+          id="comment"
+          label={t('users.dialog.comment')}
+          type="text"
+          fullWidth
+          variant="standard"
+          placeholder={t('users.dialog.comment')!!}
+          multiline
+          minRows={2}
+        />
+      </DialogContent>
+      <DialogActions>
+        <Button onClick={closeDialog}>{t('users.dialog.cancel')}</Button>
+        <Button onClick={closeDialog}>{t('users.dialog.save')}</Button>
+      </DialogActions>
+    </Dialog>
+  );
+}
+
+
+function createData(
+  role: string,
+  fullName: string,
+) {
+  return { role, fullName };
+}
+
+const rows = [
+  createData('Editor', 'Andrusov Nikita Andreevich'),
+  createData('Intern', 'Ageevn Nikolay Alexandrovich(?)'),
+  createData('Guest', 'Nebolsin Alexey Romanovich'),
+  createData('Guest', 'Nilov Lev Evgenyevich'),
+];
+
+interface BasicRowProps {
+  role: string;
+  fullName: string;
+  openDialog: (fullName: string) => void;
+}
+
+function SingleUser({ role, fullName, openDialog }: BasicRowProps) {
+  const { t } = useTranslation();
+
+  return (
+    <TableRow key={fullName}>
+      <TableCell scope="row">
+        {role}
+      </TableCell>
+      <TableCell>{fullName}</TableCell>
+      <TableCell><Button onClick={() => { openDialog(fullName) }}>{t('users.edit')}</Button></TableCell>
+    </TableRow>
+  );
+}
+
+
 function Users() {
-    return <h1>Users component</h1>
+  const { t } = useTranslation();
+  const [name, setName] = React.useState('');
+
+  const openDialog = (fullName: string) => {
+    console.log('openDialog', fullName);
+    setName(fullName);
+  };
+
+  const closeDialog = () => {
+    setName('');
+  };
+
+  return <Box display="flex" flexDirection="column" alignItems="center">
+    <h2>{t('users.title')}</h2>
+    <EditUserDialog open={name !== ''} name={name} closeDialog={closeDialog} />
+    <TableContainer component={Paper}>
+      <Table sx={{ minWidth: 650 }} aria-label="simple table">
+        <TableHead>
+          <TableRow>
+            <TableCell sx={{ 'fontWeight': 'bold' }}>{t('users.role')}</TableCell>
```

Comment:
It's better to use keyPrefix parameter in the useTranslation function then typing `'users....'` every time ypo use translation
Like this: `const { t } = useTranslation('translation', { keyPrefix: "translation.users" })`